docs/_includes/textbox.html

```diff
@@ -28,14 +28,14 @@ <h3>Readonly Textbox</h3>
     <p>Use the <span class="highlight">readonly</span> attribute to prevent any modification of the value.</p>
     <div class="demo">
         <div class="demo__inner">
-            <span class="textbox">
+            <span class="textbox textbox--readonly">
                 <input aria-label="Textbox demo" class="textbox__control" type="text" value="Readonly textbox" readonly />
             </span>
         </div>
     </div>
 
     {% highlight html %}
-<span class="textbox">
+<span class="textbox textbox--readonly">
```

Comment:
I think we should not use this for all cases but only for icon cases. 
For default we should keep disabled only on button.

Reply:
Discussed, we decided that what I have now is okay for consistency.